js/ui/drop_down_editor/ui.drop_down_editor.d.ts

```diff
@@ -125,6 +126,12 @@ export interface dxDropDownEditorOptions<TComponent> extends dxTextBoxOptions<TC
      * @public
      */
     showDropDownButton?: boolean;
+    /**
+     * @docid
+     * @default null
```

Comment:
seems like default value should not be null

Reply:
yep